### PR TITLE
Note server http body size limit

### PR DESCRIPTION
**Pull request by szokeasaurusrex**
## DESCRIBE YOUR PR
Clarify user confusion [here](https://github.com/getsentry/sentry-python/issues/3209#issuecomment-2202444443) by noting that the Sentry server enforces an limit on HTTP body size sent with an event.
 
## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/platforms/python/configuration/options.mdx [hunk 1]
```diff
@@ -172,6 +172,8 @@ This parameter controls whether integrations should capture HTTP request bodies.
 - `medium`: Medium and small requests will be captured (typically 10KB).
 - `always`: The SDK will always capture the request body as long as Sentry can make sense of it.
 
+Please note that the Sentry server [limits HTTP request body size](https://develop.sentry.dev/sdk/data-handling/#variable-size). The server **always enforces its size limit**, regardless of how you configure this option!
```
Comment (sentrivana) [inline docs/platforms/python/configuration/options.mdx:175]: Not sure if we need the bold text and the exclamation point; feels like it clashes with the usual tone of the docs a bit. Wdyt about simply leaving them out?

```suggestion
Please note that the Sentry server [limits HTTP request body size](https://develop.sentry.dev/sdk/data-handling/#variable-size). The server always enforces its size limit, regardless of how you configure this option.
```

**Review by sentrivana — APPROVED** [review]
LGTM, please consider suggestion.

**Comment by szokeasaurusrex** [comment]
@sentrivana I applied the suggestion!

**Review by vivianyentran — APPROVED** [review]
Thanks for adding this note!

**Pull request merged**